### DISCUSSION TITLE
Auto-select next keybind control when editing osu!mania keys

### DISCUSSION THREAD

**Discussion by chev2** [ppy/osu#21110] [Ideas] [answered]
osu!stable has a nice quality-of-life UX feature that auto-selects the next mania key for binding in the keyboard layout editor:

![screenshot064](https://user-images.githubusercontent.com/11602755/199862932-54b8b645-4690-4926-b5c9-cae0984bb95a.png)

Bringing this feature to the current key configurator in lazer would be really nice, so I don't have to put in my key, click the next keybind button, press the next key, etc.

![Screenshot 2022-11-03 175850](https://user-images.githubusercontent.com/11602755/199862623-732a44d2-bd2b-4025-aab2-27666f95fcf4.png)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/21270
